eoscostarica/eosio-spend-explorer
# feat(webapp): add 18 colors for delegates chart

### Loop the 18 colors for delegates

### What does this PR do?

- Resolve #86 

### Steps to test

1. run the project
2. open in your brouser localhost:3000
3. click on income in the sidebar
4. see the eigthteen static color in the chart

![image](https://user-images.githubusercontent.com/56526123/188932827-35e17ea2-c4b6-460d-9b84-3d4f50c44de4.png)

#### CheckList
